api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -196,7 +196,11 @@
 			<column name="diagnosis_id" type="int" autoIncrement="true">
 				<constraints primaryKey="true" nullable="false" />
 			</column>
+			<column name="encounter_diagnosis_coded" type="int" />
+			<column name="encounter_diagnosis_non_coded" type="varchar(255)" />
```

Comment:
Let us drop encounter_ prefix from the column names

Reply:
Done.